example/lib/main_list.dart

```diff
@@ -34,7 +34,7 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
-  static const _pageCount = 20;
+  static const _pageCount = 200;
```

Comment:
Let's not change this in the primary demo. The user shouldn't have to scroll for 200 pages to see what happens at the bottom.

If you need a dedicated example related to this behavior, then please create a new entrypoint with the desired demo.